src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pypylon.py

```diff
@@ -35,7 +35,7 @@
 # Exclude the C++-extensions from automatic search, add them manually as data files
 # their dependencies were already handled with collect_dynamic_libs
 excludedimports = ['pypylon._pylon', 'pypylon._genicam']
-for filename, module in collect_all('pypylon._pylon')[0]:
+for filename, module in collect_all('pypylon')[0]:
```

Comment:
As we are interested only in datas list (the first element of the returned tuple), this `collect_all` call should be replaced with `collect_data_files('pypylon', include_py_files=True)˙.

Reply:
You are absolutely right. Made the change.